pkg/kyverno/common/common.go

```diff
@@ -978,3 +982,35 @@ func GetKindsFromPolicy(policy *v1.ClusterPolicy) map[string]struct{} {
 	}
 	return kindOnwhichPolicyIsApplied
 }
+
+//GetPatchedResourceFromPath - get patchedResource from given path
+func GetPatchedResourceFromPath(fs billy.Filesystem, path string, isGit bool, policyResourcePath string) (unstructured.Unstructured, error) {
+	var patchedResourceBytes []byte
+	var patchedResource unstructured.Unstructured
+	var err error
+	if isGit {
+		if len(path) > 0 {
+			filep, err := fs.Open(filepath.Join(policyResourcePath, path))
+			if err != nil {
+				fmt.Printf("Unable to open patchedResource file: %s. error: %s", path, err)
+			}
+			patchedResourceBytes, err = ioutil.ReadAll(filep)
+		}
+		if err != nil {
+			fmt.Printf("\n----------------------------------------------------------------------\nfailed to load patchedResource: %s. \nerror: %s\n----------------------------------------------------------------------\n", path, err)
+		}
+		patchedResource, err = GetPatchedResource(patchedResourceBytes)
+		if err != nil {
+			return patchedResource, err
+		}
+	} else {
+		patchedResourceBytes, err = getFileBytes(path)
+		if err != nil {
+			fmt.Printf("\n----------------------------------------------------------------------\nfailed to load patchedResource: %s. \nerror: %s\n----------------------------------------------------------------------\n", path, err)
```

Comment:
I see this print statement is duplicated multiple times. Can this be converted into a method and `path` and `error` can be passed to it ?

Reply:
> 
> 
> I see this print statement is duplicated multiple times. Can this be converted into a method and `path` and `error` can be passed to it ?

What if I create this function, and replace all multiple print statement with this function.
```
func FailToFetchPatchedResource(path string, err error) {
	fmt.Printf("\n----------------------------------------------------------------------\nfailed to fetch patchedResource: %s. \nerror: %s\n----------------------------------------------------------------------\n", path, err)
}
```

Reply:
that is what I said